builtins/amp-img.js

```diff
@@ -167,6 +177,31 @@ export class AmpImg extends BaseElement {
     return true;
   }
 
+  /**
+   * Checks to see if there is a placeholder that is blurred
```

Comment:
Be more specific. "If a blurry placeholder exists, hides the child <img> until it's fully loaded."